hc/local_settings.py

```diff
@@ -0,0 +1,9 @@
+import os
+import dj_database_url
+
+# If deploying to heroku, the database config will be populated automatically
+# Create an environment variable HEROKU_DEPLOYMENT in heroku and set it to 1
+if os.environ.get("HEROKU_DEPLOYMENT") == '1':
+    DATABASES = {
```

Comment:
I guess this can be done in the `hc/settings.py` file. You just need to reference it correctly.

Reply:
Noted and will resolve.